connection.go

```diff
@@ -1009,3 +1009,13 @@ func (conn *Connection) OverrideSchema(s *Schema) {
 		conn.Schema = s
 	}
 }
+
+// Prepare passes a sql statement to Tarantool for preparation synchronously.
+func (conn *Connection) Prepare(expr string) (*PreparedStatement, error) {
```

Comment:
`Prepare` -> `NewPrepareStatement`

Two reasons:
1. We can create a `Prepare` request in the future if we want.
2. The naming will be synced with #185  ( `NewStream`).

Reply:
Fixed.

Reply:
Oh, typo `NewPrepareStatement` -> `NewPreparedStatement`. I'm sorry.